### PR TITLE
[examples] Updated Nuxt starter to v3 stable

### PR DESCRIPTION
**Pull request by steven-tey**
CleanShot 2022-11-16 at 17 13 02

### CODE REVIEW

File: examples/nuxtjs/README.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
-# Nuxt.js Example
+# Nuxt 3 Minimal Starter
```
Comment (leerob) [inline examples/nuxtjs/README.md]:
```suggestion
# Nuxt Example

Deploy your [Nuxt](https://nuxt.com) project to Vercel with zero configuration.

[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/vercel/tree/main/examples/nuxtjs&template=nuxtjs)

_Live Example: https://nuxtjs-template.vercel.app_
```

File: examples/nuxtjs/README.md [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+# Nuxt Example
 
-Deploy your [Nuxt.js](https://nuxtjs.org) project to Vercel with zero configuration.
+Deploy your [Nuxt](https://nuxt.com) project to Vercel with zero configuration.
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/vercel/tree/main/examples/nuxtjs&template=nuxtjs)
 
 _Live Example: https://nuxtjs-template.vercel.app_
 
+Look at the [nuxt 3 documentation](https://v3.nuxtjs.org) to learn more.
```
Comment (leerob) [inline examples/nuxtjs/README.md]:
```suggestion
Look at the [Nuxt 3 documentation](https://v3.nuxtjs.org) to learn more.
```

File: examples/nuxtjs/package.json [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-    "start": "node .output/server/index.mjs"
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare"
   },
   "devDependencies": {
-    "nuxt3": "latest"
+    "nuxt": "3.0.0"
```
Comment (leerob) [inline examples/nuxtjs/package.json]:
```suggestion
    "nuxt": "^3.0.0"
```

**Review by leerob — CHANGES_REQUESTED** [review]
Can we use pnpm please?

**Comment by steven-tey** [comment]
@leerob done!

**Review by leerob — APPROVED** [review]

**Pull request merged**